collection/ebogerudlan.js

```diff
@@ -0,0 +1,20 @@
+import SimpleSchema from 'simpl-schema';
+
+Eboger = new Meteor.Collection('eboger');
+
+EbogerSchema = new SimpleSchema(
+  {
+    desc: {
+      type: String
+    },
+
+    year: {
+      type: Number
+    },
+
+    total: {
+      type: Number
+    }
+  });
+
+  Eboger.attachSchema(EbogerSchema);
```

Comment:
Hmm... really confused about the naming here.

A collection named eboeger is way to specific. We need a general mongo schema for persisting data-sets,

I think what you should do here instead, is to define our collection that will house our data-sets. So perhaps:

1. Make a collection/dataset.js
2. Make new DatasetSchema instead.
3. Perhaps make a dataset.html which contains a template that will render our data-sets (as  table)